tests/Unit/Finder/FinderTest.php

```diff
@@ -91,4 +112,25 @@ private function createFiles(Path ...$files): void
                 ->dumpFile($path->toString(), $content);
         }
     }
+
+    /**
+     * @test
+     */
+    public function findFilesInSymlinkedFolder(): void
```

Comment:
Actually you could prepare directory structure (for example in "tests/resources/fixtures/") in `tests` directory, with this change code will be simpler and there is no need for `Filesystem::symlink` method.

Reply:
Sorry for the delay. Did just that.